test/functional/helpers/constants/domain.js

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export const THIRD_PARTY_DOMAIN = "edge.adobedc.net";
+export const FIRST_PARTY_DOMAIN = "firstparty.alloyio.com";
```

Comment:
We had a question yesterday from the Konductor team if they could run our tests against a locally running version of Konductor. @shammowla I think we could allow these domains to be set via a environment variable

Reply:
Yes pls :)

Reply:
It looks like they want a custom port and no edge base path as well. I'm going to submit a separate PR.

Reply:
Oh and they want to run on http instead of https. Some of this will require changes to source code.